odxtools/utils.py

```diff
@@ -11,15 +11,9 @@ def read_description_from_odx(et_element: ElementTree.Element):
     #       This just represents it as XHTML string. 
     if et_element is None:
         return None
+ 
+    raw_string = (et_element.text or '') + ''.join(ElementTree.tostring(e, encoding='unicode') for e in et_element)
```

Comment:
hm, I guess this is a good change, but I think that statement is quite hard to read and the `or` in `(et_element.text or '')` is quite superfluous. would you mind changing it to something like this?
```suggestion
    raw_string = et_element.text
    for e in et_element:
        raw_string += ElementTree.tostring(e, encoding='unicode')
```

Reply:
` or ''` is required since `et_element.text` could be None, and concatenation of None and a string is not valid

Reply:
Ok, fair enough. I just thought that all ElementTree nodes come with a `.text` attribute. maybe not...

Reply:
Ah, I have a suspicion: The type checker probably declares the `.text` attribute as `Optional[str]` because for tags without content like `<FOO />` it is (probably) `None`. I suppose not having the `or ''` would thus not bite at runtime but I think it is good to have it to keep `mypy` happy...